docs/data/data-grid/aggregation/aggregation.md

```diff
@@ -46,98 +46,99 @@ This will disable all features related to aggregation, even if a model is provid
 
 {{"demo": "AggregationDisabled.js", "bg": "inline", "defaultCodeOpen": false}}
 
-### For some columns
+### For specific columns
 
 To disable aggregation on a specific column, set the `aggregable` property on its column definition (`GridColDef`) to `false`.
 
 In the example below, the **Year** column is not aggregable since its `aggregable` property is set to `false`.
 
 {{"demo": "AggregationColDefAggregable.js", "bg": "inline", "defaultCodeOpen": false}}
 
-### Aggregating non-aggregable columns programmatically
+## Aggregating non-aggregable columns
 
-To apply aggregation programmatically on non-aggregable columns (columns with `aggregable: false` in the [column definition](/x/api/data-grid/grid-col-def/)), you can provide aggregation model in one of the following ways:
+To apply aggregation programmatically on non-aggregable columns (columns with `aggregable: false` in the [column definition](/x/api/data-grid/grid-col-def/)), you can provide the aggregation model in one of the following ways:
 
-1. Pass `aggregation.model` to the `initialState` prop. This will [initialize the aggregation](/x/react-data-grid/aggregation/#initialize-aggregation) with the provided model.
-2. Provide the `aggregationModel` prop. This will [control the aggregation](/x/react-data-grid/aggregation/#controlled-aggregation) with the provided model.
-3. Call the API method `setAggregationModel`. This will set the aggregation with the provided model.
+- Pass `aggregation.model` to the `initialState` prop. This initializes aggregation with the provided model.
+- Provide the `aggregationModel` prop. This controls aggregation with the provided model.
+- Call the API method `setAggregationModel()`. This sets the aggregation with the provided model.
```

Comment:
sets aggregation sounds a bit weird
could we maybe use a different term?

Suggestion:
```suggestion
- Call the API method `setAggregationModel()`. This applies an aggregation with the provided model.
```